pacai/ui/pacmanDisplay.py

```diff
@@ -0,0 +1,663 @@
+    def drawPane(self):
+        pass
+
+    @abstractmethod
+    def updateScore(self, score):
+        pass
+
+    def __init__(self, layout, gridSize):
+        self.gridSize = gridSize
+        self.width = layout.width * gridSize
+        self.base = (layout.height + 1) * gridSize
+        self.height = graphicsConstants.INFO_PANE_HEIGHT
+        self.fontSize = graphicsConstants.DEFAULT_PANE_FONT_SIZE
+        self.textColor = graphicsConstants.PACMAN_COLOR
+
+    def toScreen(self, position, y = None):
+        """
+        Translates a point relative from the bottom left of the info pane.
+        """
+
+        if y is None:
+            x, y = position
+        else:
+            x = position
+
+        x = self.gridSize + x  # Margin
+        y = self.base + y
+        return x, y
+
+    def initializeGhostDistances(self, distances):
+        self.ghostDistanceText = []
+
+        size = graphicsConstants.DEFAULT_TEXT_SIZE
+        if self.width < 240:
+            size = 12
+        if self.width < 160:
+            size = 10
+
+        for i, d in enumerate(distances):
+            t = graphicsUtils.text(self.toScreen(self.width / 2 + self.width / 8 * i, 0),
+                    graphicsConstants.GHOST_COLORS[i + 1], d, graphicsConstants.TIMES_FONT, size,
+                    graphicsConstants.TEXT_MOD_BOLD)
+            self.ghostDistanceText.append(t)
+
+    def setTeam(self, isBlue):
+        if isBlue:
+            text = graphicsConstants.TEAM_BLUE
+        else:
+            text = graphicsConstants.TEAM_RED
+        self.teamText = graphicsUtils.text(self.toScreen(300, 0), self.textColor, text,
+                graphicsConstants.TIMES_FONT, self.fontSize, graphicsConstants.TEXT_MOD_BOLD)
+
+    def updateGhostDistances(self, distances):
+        if len(distances) == 0:
+            return
+
+        if 'ghostDistanceText' not in dir(self):
+            self.initializeGhostDistances(distances)
+        else:
+            for index, ghostDistance in enumerate(distances):
+                graphicsUtils.changeText(self.ghostDistanceText[index], ghostDistance)
+
+    def drawGhost(self):
+        pass
+
+    def drawPacman(self):
+        pass
+
+    def drawWarning(self):
+        pass
+
+    def clearIcon(self):
+        pass
+
+    def updateMessage(self, message):
+        pass
+
+    def clearMessage(self):
+        pass
+
+class AbstractPacmanGraphics(ABC):
+    @abstractmethod
+    def startGraphics(self, state):
+        pass
+
+    @abstractmethod
+    def update(self, state):
+        pass
+
+    def __init__(self, zoom = graphicsConstants.DEFAULT_ZOOM,
+            frameTime = graphicsConstants.DEFAULT_FRAME_TIME,
+            capture = graphicsConstants.DEFAULT_CAPTURE_ARG,
+            gif = graphicsConstants.DEFAULT_GIF_ARG,
+            gifSkipFrames = graphicsConstants.DEFAULT_GIF_FRAME_SKIP,
+            gifFPS = graphicsConstants.DEFAULT_GIF_FPS):
+        self.have_window = 0  # Not used anywhere.
```

Comment:
Here we should probably do haveWindow to keep consistent formatting.

Reply:
Shouldn't I just remove it since it is not used anywhere?